docs/src/tool/index.md

```diff
@@ -105,24 +109,30 @@ named `default`.
 + The environment variable `REACH_DEBUG`, if set to any non-empty value, enables debug messages from the Reach compiler, which will appear in the console.
   This debug information includes: the estimated cost of the contract on Algorand.
   This variable automatically enabled `--intermediate-files`.
+  
+  A `reach compile` usage example is available in [Overview](##over-compile). 
+
+# {#ref-usage-init} `reach init`
 
-## {#ref-usage-init} `reach init`
+Running `reach init` creates the `index.rsh` and `index.mjs` template files required for a basic Reach DApp. This allows you to open the files and start writing code. The `index.rsh` file is the DApp and is written in Reach, and the `index.mjs` file is the frontend of the DApp and is written in JavaScript.
 
-You can create template `index.rsh` and `index.mjs` files for a simple Reach app by running
+To run `reach init`, use the following command: 
 
 ```cmd
-$ reach init
+$ ./reach init
 ```
 
 # {#ref-usage-run} `reach run`
 
-You can run a simple Reach application by executing
+The `reach run` command with no argumenst starts the `index` application in the current directory by default, but you can set a different directory, application name, or both.
+
+The `reach run` command uses the following syntax:
```

Comment:
I think a purist could prove me wrong but, (programmer-)colloquially-speaking, syntax doesn't feel like the right word here... Maybe "interface" or "exposes these options"?

Reply:
How about structure?

Reply:
Structure sounds less right than syntax to me. Those aren't terms that programmers/console dorks like me use in this context.

Reply:
The `reach run` command uses the following interface: Is this proper usage?

Reply:
Yup, I think sounds more natural, but if I'm nitpicking I'd probably say "supports" or "exposes" "_ the following interface".